docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,28 @@
+# Troubleshooting
+If you're reading this, something went wrong.
+Don't worry though, as this is the most thorough guide to help you ~~(and to help us)~~!
+
+## `./Impostor.Server: line 1: ELF: not found` (plus other errors)
+No idea where you got that system. But we clearly do **NOT** support it.
+
+## `cannot execute binary file: Exec format error`
+Please check that you have downloaded the right version of Impostor, as we mantain two CPU architectures (x64 and ARM).  
+Unless you are running Impostor on a SBC (Single-Board Computer), like the Raspberry Pi, you most likely want to use the x64 version.
+
+## `./Impostor.Server: Permission denied`
+This is an error related to Linux file permissions.  
+You can solve this by doing: `chmod +x Impostor.Server`
+_(Note: only run `chmod` when you trust the software. Linux blocks downloaded executables by default for a reason)_
```

Comment:
More because when you download it, it doesn't have the executable bit set. You can download files that have this already set.

Reply:
Do I really need to specify that? (are we trying to keep this a blob of text that people will most likely ignore)
Generally files dont hold their permissions

Reply:
No, I'm just saying you are putting inaccurate information here.

Reply:
Fixed